### ISSUE TITLE
erro

### ISSUE THREAD

**Issue by wstroks** [godot-extended-libraries/godot-tensorflow-workspace#1]
I'm having this error when I compile

scons -j8 platform=x11

71%] scons: *** [thirdparty/tensorflow/tensorflow/lite/nnapi_delegate.x11.tools.64.o] Source thirdparty/tensorflow/tensorflow/lite/nnapi_delegate.cc' not found, needed by target thirdparty/tensorflow/tensorflow/lite/nnapi_delegate.x11.tools.64.o'.

I tried everything but I can not solve!!!

**Comment by fire** [comment]
Does thirdparty/tensorflow/ exist in your godot directory? 

Did you use depot_tools to fetch the folders?

**Comment by wstroks** [comment]
yes, I tried to compile on linux not getting it !! I changed it to windows and it worked ... thank you very much it will help me a lot <3 !! With time I will try to contribute some games here haha

**Comment by fire** [comment]
I haven't tested it on Linux yet :(

**Comment by wstroks** [comment]
Calm hehe, just a doubt. The importing model in godot did you do as? I created a keras model and export tflite and it does not work right ... could it explain how you did it? I used categorical accuracy (CNN)

**Comment by gylleus** [comment]
Same issue here on Ubuntu 16.04. Just followed the steps in the README:

```bash
git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
git clone https://github.com/godot-extended-libraries/godot-tensorflow-workspace
cd godot-tensorflow-workspace
PATH=$PATH:../depot_tools gclient sync
cd src/godot
scons platform=x11 tools=yes -j8

...

[Initial build] scons: *** [thirdparty/tensorflow/tensorflow/lite/nnapi_delegate.x11.tools.64.o] Source `thirdparty/tensorflow/tensorflow/lite/nnapi_delegate.cc' not found, needed by target `thirdparty/tensorflow/tensorflow/lite/nnapi_delegate.x11.tools.64.o'.
scons: building terminated because of errors.
```

**Comment by fire** [comment]
I think there is a missing conditional include for Linux.

Will look into it when I have bandwidth.

The problem is around here: https://github.com/godot-extended-libraries/godot-tensorflow/blob/master/SCsub#L148

**Comment by fire** [comment]
I updated for Linux, please check. Version was also changed to Tensorflow v2.0.0.
